samza-kafka/src/main/scala/org/apache/samza/system/kafka/KafkaSystemConsumer.scala

```diff
@@ -193,8 +193,8 @@ private[kafka] class KafkaSystemConsumer(
 
         // addTopicPartition one at a time, leaving the to-be-done list intact in case of exceptions.
         // This avoids trying to re-add the same topic partition repeatedly
-        def refresh(tp: List[TopicAndPartition]) = {
-          val head :: rest = tpToRefresh
+        def refresh() = {
```

Comment:
Why is this change required?

Reply:
It doesn't compile with scala 2.10 in unit tests. Specifically, the scalaTest version we are using ("3.0.1") has some compatibility issues with 2.10.